webapp/src/main/java/com/box/l10n/mojito/rest/security/PasswordChangeRequest.java

```diff
@@ -0,0 +1,6 @@
+package com.box.l10n.mojito.rest.security;
+
+import java.io.Serializable;
+
+public record PasswordChangeRequest(String currentPassword, String newPassword)
+    implements Serializable {}
```

Comment:
For simplicity sake I'd remove "implements Serializable", record do implement it anyway.

Reply:
changed it.